### PR TITLE
miscellaneous cosmetic fixes

### PR DESCRIPTION
**Pull request by llyyr**
Mostly to silence clangd, but the sd_ass integer division is actually a bug fix for subtitles with unknown durations

### CODE REVIEW

File: common/msg_control.h [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #ifndef MP_MSG_CONTROL_H
 #define MP_MSG_CONTROL_H
 
+#include "common/msg.h"
```
Comment (Traneptora) [inline common/msg_control.h:4]: standard headers go above custom headers

File: filters/f_async_queue.h [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "filter.h"
+#include <stdint.h>
```
Comment (Traneptora) [inline filters/f_async_queue.h:4]: standard headers should go above custom ones

**Comment by Traneptora** [comment]
What does forward-declaring mp_log accomplish, exactly?

**Comment by llyyr** [comment]
> What does forward-declaring mp_log accomplish, exactly?

"Declaration of [...] will not be visible outside of this function" is the clang warning, I think this is purely cosmetic since mp_log ends up being included.

File: video/out/filter_kernels.c [hunk 3]
```diff
@@ -134,7 +134,7 @@ static void mp_compute_weights(struct filter_kernel *filter, double f,
     assert(filter->size > 0);
     double sum = 0;
     for (int n = 0; n < filter->size; n++) {
-        double x = f - (n - filter->size / 2 + 1);
+        double x = f - (n - filter->size / 2.0 + 1);
```
Comment (haasn) [inline video/out/filter_kernels.c:137]: This will be incorrect if `filter->size` is ever an odd number.

**Comment by haasn** [comment]
PR title is misleading, changing integer division to floating point division is NOT a cosmetic change. The integer truncation (i.e., rounding down) may have been an intentional feature of the original code.

**Comment by llyyr** [comment]
I'll drop the integer division changes for now and thoroughly test if they were intended or not. I'm pretty sure the sub one at least is a bug fix, but I'll do it in another PR later.

File: sub/ass_mp.h [hunk 4]
```diff
@@ -26,9 +26,11 @@
 #include <ass/ass.h>
 #include <ass/ass_types.h>
 
-// This is probably arbitrary.
-// sd_lavc_conv might indirectly still assume this PlayResY, though.
+// These PlayResX and PlayResY values are arbitrary and taken from lavc.
+// lavc assumes these values when converting to ass generally. Moreover, these
+// values are also used by default in VSFilter, so it isn't that arbitrary.
 #define MP_ASS_FONT_PLAYRESY 288
+#define MP_ASS_FONT_PLAYRESX 384
```
Comment (Dudemanguy) [inline sub/ass_mp.h:33]: Sorry really pointless nit but could you put X before Y? I keep reading this backwards.

**Review by Dudemanguy — APPROVED** [review]

**Pull request merged**